commands.py

```diff
@@ -277,3 +277,212 @@ def run_discord_bot():
+            click.echo('  Chunk start: {}'.format(start))
+
+            players = scraper.get_players(database, sort=players_sort, start=start, limit=chunks, basic=True)
+
+            if not players:
+                click.secho('No more players to handle', fg='green')
+
+                break
+
+            all_player_names = [player.username for player in players]
+
+            existing_rwr_accounts = RwrAccount.query.filter(
+                RwrAccount.type == rwr_account_type,
+                RwrAccount.username.in_(all_player_names)
+            ).all()
+
+            rwr_accounts_by_username = {rwr_account.username: rwr_account for rwr_account in existing_rwr_accounts}
+
+            # Remove players already having an RWR account and that are already up-to-date in the DB to prevent saving duplicate stats
+            for player in players:
+                if player.username in rwr_accounts_by_username and rwr_accounts_by_username[player.username].updated_at.floor('day') >= now:
+                    players.remove(player)
+
+            # Create RWR accounts if they do not exists / touch the updated_at timestamp if they exists
+            for player in players:
+                if player.username not in rwr_accounts_by_username:
+                    rwr_account = RwrAccount()
+
+                    rwr_account.username = player.username
+                    rwr_account.type = rwr_account_type
+
+                    rwr_accounts_by_username[player.username] = rwr_account
+                else:
+                    rwr_account = rwr_accounts_by_username[player.username]
+
+                    rwr_account.updated_at = arrow.utcnow().floor('minute')
+
+                db.session.add(rwr_account)
+
+            db.session.commit()
+
+            # Save current stats of the RWR accounts (if their respective stats changed)
+            for player in players:
+                rwr_account_stat = RwrAccountStat()
+
+                rwr_account_stat.leaderboard_position = player.position
+                rwr_account_stat.xp = player.xp
+                rwr_account_stat.kills = player.kills
+                rwr_account_stat.deaths = player.deaths
+                rwr_account_stat.time_played = player.time_played
+                rwr_account_stat.longest_kill_streak = player.longest_kill_streak
+                rwr_account_stat.targets_destroyed = player.targets_destroyed
+                rwr_account_stat.vehicles_destroyed = player.vehicles_destroyed
+                rwr_account_stat.soldiers_healed = player.soldiers_healed
+                rwr_account_stat.teamkills = player.teamkills
+                rwr_account_stat.distance_moved = player.distance_moved
+                rwr_account_stat.shots_fired = player.shots_fired
+                rwr_account_stat.throwables_thrown = player.throwables_thrown
+                rwr_account_stat.rwr_account_id = rwr_accounts_by_username[player.username].id
+
+                if rwr_account_stat.can_be_saved:
```

Comment:
Change logic here to save a big number of DB queries: create all the players `RwrAccountStat` objects, then query one time the DB to check with all the hashes.